src/workerd/jsg/setup.h

```diff
@@ -365,7 +365,7 @@ class Isolate: public IsolateBase {
   bool serialize(
       Lock& js, std::type_index type, jsg::Object& instance, Serializer& serializer) override {
     KJ_IF_SOME(func, wrapper->serializerMap.find(type)) {
-      func(js, instance, serializer);
+      func(*wrapper, js, instance, serializer);
```

Comment:
minor nit: can we keep the `js` argument first per our usual convention. It's fine if not but I definitely prefer the consistency

Reply:
Well, in many internal parts of JSG, the convention is that `TypeWrapper&` is the first argument, so there's a bit of a conflict. `DeserializeFunc` actually already had `TypeWrapper&` coming first, so in this commit when I added a similar `TypeWraper&` argument to `SerializeFunc`, I made it match.

These are entirely internal functions so I don't think it's a big deal one way or the other.